spec.md

```diff
@@ -559,10 +559,53 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+Registries MAY support an enhanced tag listing experience when clients pass an `Accept` header value of `application/vnd.oci.image.index.v1+json`.
+If a registry supports this mode, they MUST set the `Content-Type: application/vnd.oci.image.index.v1+json` header.
+Registries supporting the enhanced tag list MUST support these additional query parameters:
+* `sort_by` - allowed values are `created_at`, `name`
+* `sort_order` - allowed values are `asc` and `desc`
+
+The response MUST be a json body in the following format:
```

Comment:
Just a note: This example response looks like an `index.json` in the OCI image layout, which serves as a tag list or generally a manifest list.

Reply:
That's entirely intentional, and even why it's reusing the annotation that's "only for `index.json`" 😅

Reusing the "image index" structure for this also matches the design of the referrers API, which is a useful pattern because it avoids defining "yet another data schema" without a really strong reason.